clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -117,6 +121,77 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(config_path):
```

Comment:
Change the method name to `_parse_configuration` to indicate this is a private method. Also, there is no need for this to be a nested method. Just make it a normal method for the object.

Reply:
Done